ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
Summary,
- `core.js` must not list dependency on `core.css` or on any other CSS.
- The widgets will list its dependency on `core.css` themselves, e.g., dialog.js: `//>>css.structure: base/core.css, base/dialog.css`.
- There is no agreement on how button CSS should be defined yet.

All true?